storage/src/vespa/storage/common/bucket_stripe_utils.h

```diff
@@ -22,7 +22,14 @@ size_t stripe_of_bucket_key(uint64_t key, uint8_t n_stripe_bits) noexcept;
  *
  * This also asserts that the number of stripes is valid (power of 2 and within MaxStripes boundary).
  */
-uint8_t calc_num_stripe_bits(size_t n_stripes) noexcept;
+uint8_t calc_num_stripe_bits(uint32_t n_stripes) noexcept;
+
+/**
+ * Validates the number of stripes and returns the (potentially adjusted) value.
+ *
+ * This ensures the number of stripes is a power of 2 and within MaxStripes boundary.
+ */
+uint32_t validate_num_stripes(uint32_t n_stripes) noexcept;
```

Comment:
To me, the prefix "validate" tends to imply that the function performs some explicit action if the validation fails (such as throwing an exception). This function communicates that this is not the case by being tagged `noexcept`, but I wonder if there's a more fitting name for it still. Perhaps just `adjusted_num_stripes` ref. the comment?

Reply:
I agree. Fixed.